### ISSUE TITLE
Plate summaries

### ISSUE THREAD

**Issue by MatthewMah** [DReichLab/waldo#1]
Table of ALL info for each sample at this stage. For captures, this would include info from the sample prep queue, powder batch, lysate batch, extract batch, library batch, and capture or raw batch for each library on the plate.

**Comment by JNWorkman** [comment]
I'm unable to add a label, but would like to put this in the urgent category

**Comment by JNWorkman** [comment]
Including all info from powder batches on plates is especially important, as that isn't accessible except by going to each separate powder batch at the moment.

**Comment by MatthewMah** [comment]
@KimCallan @JNWorkman How did you want these additional fields to appear? Include on all existing spreadsheets? Provide a second spreadsheet option that includes the additional fields?

**Comment by JNWorkman** [comment]
I think it would be preferable to have a second spreadsheet that contains all additional information and leave the existing spreadsheets that are used for updating each stage as is.

**Comment by KimCallan** [comment]
I agree with Noah about having a second spreadsheet kept apart from our uploading sheets

**Comment by MatthewMah** [comment]
This depends on completion of #57

**Comment by KimCallan** [comment]
At some point we would also like to have all the SQ report info added in that comes from Adams pipeline. And have some columns in there that are programmed to do some math using the SQ columns
